src/apiClient.js

```diff
@@ -7,15 +7,22 @@ const reportRateLimits = {
     volumechange: 3000
 };
 
+/** Approximate LAN bitrate */
+const LAN_BITRATE = 140000000;
```

Comment:
I think this should be a bitrate, not a "byte"-rate. Most people should have at least Gbit LAN nowadays.

```suggestion
const LAN_BITRATE = 1000000000;
```

Reply:
This is 140Mbps from https://github.com/jellyfin-archive/jellyfin-apiclient-javascript/blob/6e3ace3aac1228870fd85b1af41502c3bb688e6d/src/apiClient.js#L4093

I've never reached clear 1Gbps - only ~700+Mbps.

Reply:
Right, but those 140 Mbps seem to be chosen at random. A slightly higher minimum for the maximum bitrate might be appropriate nowadays.

Reply:
Someone can use an older TV with 100M Ethernet or 2.4GHz WiFi. We are still trying to support legacy devices (obviously, we can't do this forever).

I think 140 was chosen because it further scales to 70% (140*0.7 = 98):
https://github.com/jellyfin-archive/jellyfin-apiclient-javascript/blob/6e3ace3aac1228870fd85b1af41502c3bb688e6d/src/apiClient.js#L4056

Reply:
Hmm, I guess that makes sense. Didn't really think about wireless devices for this use case.

Reply:
True, but I recently came across a file as high as 400 Mbps, and I think it's ok to future proof things in that regard.